demo/index.html

```diff
@@ -17,46 +17,33 @@
 
   <title>paper-fab demo</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script type="module" src="../../paper-styles/color.js"></script>
-  <script type="module" src="../../paper-styles/default-theme.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="../../iron-icons/iron-icons.js"></script>
-  <script type="module" src="../paper-fab.js"></script>
-
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style>
-    <style is="custom-style" include="demo-pages-shared-styles">
-      paper-fab {
-        margin-left: 10px;
-        margin-right: 10px;
-      }
-
-      .vertical-section-container {
-        max-width: 550px;
-      }
-    </style>
-  </custom-style>`;
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+  <script type="module">
+    import '@polymer/paper-styles/color.js';
+    import '@polymer/paper-styles/default-theme.js';
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import '@polymer/iron-icons/iron-icons.js';
+    import '../paper-fab.js';
+    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+    const template = html`
+      <custom-style>
```

Comment:
Is multi-line formatting of template literals in HTML with webmat stable yet? My workaround is to have include style modules in one line, then the rest of the styles in HTML:

```html
  <script type="module">
    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
    const template = html`<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
    document.body.appendChild(template.content);
  </script>
  <custom-style>
    <style is="custom-style">
      paper-fab {
        /* ... */
      }
    </style>
  </custom-style>
```

cc @e111077

Reply:
im confused about what you mean by "the rest of the styles" -- you still have custom-style in the demo -- is that one fine standalone? 

also ive been multilining these styles for all my other elements and it "seems ok"? dunno.

Reply:
Hola, keanu, you may want to blow away your node_modules and package-lock.json. I have made webmat so that it should not touch the contents of a template literal at all. It will only format the first line. This also means that if all of the other contents of a script tag were tabbed by webmat, then you will have to manually tab the contents of the template literal as tabbing it will modify the value of the template literal.

Reply:
e.g.

```html
<body>
  <script>
const t = html`
  <div>foo bar</div>
`;
document.body.appendChild(t.contents);
  </script>
</body>
```

will be formatted to

```html
<body>
  <script>
	const t = html`
  <div>foo bar</div>
`;
	document.body.appendChild(t.contents);
  </script>
</body>
```

because it won't touch the contents